src/o365/spfx/commands/project/project-rename.spec.ts

```diff
@@ -106,9 +106,21 @@ describe(commands.PROJECT_RENAME, () => {
   });
 
   it('updates only the files found and skips other files', (done) => {
+    sinon.stub(command as any, 'getProjectRoot').callsFake(_ => path.join(process.cwd(), projectPath));
+    sinon.stub(command as any, 'getProject').callsFake(_ => {
+      return {
+        path: projectPath,
+        packageJson: {
+          dependencies: {},
+          name: projectPath.split('/').pop()
```

Comment:
Rather than using fixed `/` we should use OS-specific path separator (`path.sep`). In this particular case, to get the file-/directory name from the path, it would be better to use `path.basename` (https://nodejs.org/api/path.html#path_path_basename_path_ext)